CMakeLists.txt

```diff
@@ -479,6 +479,25 @@ ELSE()
 ENDIF()
 
 
+# Find the libdap package so that Aspect can connect to the OPeNDAP servers
+# Author: Kodi Neumiller
+# Added: 10/16/18
+IF(LIBDAP)
+  LIST(APPEND CMAKE_MODULE_PATH "${CMAKE_MODULE_PATH}/cmake/modules")
+  MESSAGE(STATUS "cmake module path = ${CMAKE_MODULE_PATH}")
+  FIND_PACKAGE(LIBDAP)
+  IF(${LIBDAP_FOUND})
+    FIND_PACKAGE(LibXml2)
+    FIND_PACKAGE(CURL)
```

Comment:
Same here. Also, what if one of these checks fails?

Reply:
Right now if the FIND_PACKAGE fails it just returns: "Could not find a package configuration file provided by CURL/LibXml2".

However it looks like it still finds the libdap libraries if they are installed. Without CURL or libxml2 then the code we added to utilities.cc in url_reader won't be able to read data from a url. I could add a warning or an error if one or both of the CURL/libxml2 packages aren't found.

Reply:
Then it is probably best to find the packages first and only enable it if you find all three. I still confused though, shouldn't libdab provide you with all information including what libraries we need to link to? This can be handled inside FindLIBDAP if needed.
I just checked my installation of libdab and found ``./bin/dap-config --libs``, which returns the complete link line. That would be one way to find the necessary information.

Reply:
not exactly the same as curl doesn't have other libs as dependencies, but this might help: https://github.com/Kitware/CMake/blob/master/Modules/FindCURL.cmake#L136

Reply:
I've updated the code to not build if one or both CURL and libxml2 are not found. In your latest comment are you saying I could add the dependent library lookups to FindLIBDAP?

Thanks for all of the help by the way